WISE-Community/WISE-Client
# build(NPM): Fix vulnerabilities and clean up dependencies

## Changes
- Fix 1 critical by upgrading custom-webpack dep
- Move cssnano to devDependency
- Upgrade jQuery dep
- Remove old npm commands that use concurrently, and remove concurrently
  dep (these commands are no longer used since we switched to Docker)
- Remove unused cheerio dep
- Before: 111 vulnerabilities (1 low, 80 moderate, 29 high, 1 critical).
  After: 67 vulnerabilities (55 moderate, 12 high)

## Test
- Stop docker dev containers
- Change WISE-Docker-Dev's wise-client and wise-api services to use the "latest" tag if not already
- Start docker dev containers
- Make sure that everything works as before
   - ```gulp compile-sass``` should compile sass
   - VLE, AT, CM load and work as before

Closes #488
